_posts/2025-03-31-mmaler-blogpost-1-intro.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: 'Optimizing Java for the Cloud-Native Era with Quarkus'
+date: 2025-03-31
+tags: introduction development reactive performance java cloud
```

Comment:
just pick one or two tags. the tag UI gets overloaded with all these. yes, problem in others too and on my todo to get cleaned up ;)

Reply:
I reduced it to these:introduction development performance